packages/case-study-slider/index.test.js

```diff
@@ -1,4 +1,4 @@
-import { render } from '@testing-library/react'
+import { render, screen } from '@testing-library/react'
```

Comment:
Had to refactor tests here. Added `getByTestId` in some places, to backfill tests that previously used `querySelector` with classNames. (While we could continue doing this due to how we load CSS module files in Jest, using `getByTestId` felt more appropriate).